api/comment_threads.rb

```diff
@@ -105,3 +105,16 @@
     course_stats.to_json
   end
 end
+
+get "#{APIPREFIX}/threads/:thread_id/num_followers" do |thread_id|
+  begin
```

Comment:
@chrisndodge I'm not quite sure, but most likely this `begin`-`end` pair is not needed here as `do` already creates a block. But my ruby is far from perfect, so I might be wrong.